src/Functions/test/Functions.Scenario.Tests.ps1

```diff
@@ -7,36 +7,45 @@ while(-not $mockingPath) {
 . ($mockingPath | Select-Object -First 1).FullName
 
 # Test variables
-$testSubscriptionId = "52d8cf1b-bcac-493a-bbae-f234b5ff38b0"
+$testSubscriptionId = "07308f04-ea00-494b-b320-690df74b1ce6"
 
 $resourceGroupNameWindowsPremium = "Func99-West-Europe-Win-Premium"
 $resourceGroupNameLinuxPremium = "Func99-West-Europe-Linux-Premium"
 $resourceGroupNameLinuxConsumption = "Func99-Southeast-Asia-Linux-Consumption"
 $resourceGroupNameWindowsConsumption = "Func99-Central-US-Windows-Consumption"
+$resourceGroupNameLinuxConsumptionDotNet = "Func99-Central-US-Linux-Consumption"
 
 $premiumPlanLocation = "West Europe"
 $linuxConsumptionLocation = "Southeast Asia"
+$linuxConsumptionDotNetLocation ="Central US"
 $windowsConsumptionLocation = "Central US"
 $planNameWorkerTypeLinux = "Func99-West-Europe-Linux-Premium"
 $planNameWorkerTypeWindows = "Func99-West-Europe-Windows-Premium"
 $storageAccountWindows = "functionswinstorage888"
 $storageAccountLinux = "functionslinuxstorage888"
+$userIdentityId = "/subscriptions/07308f04-ea00-494b-b320-690df74b1ce6/resourcegroups/Func99-West-Europe-Win-Premium/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ID1"
```

Comment:
ideally, these variable would go in the env file, and be set up in your 'setup' script

Reply:
Fixed.